documents/ANDROID.md

```diff
@@ -94,3 +95,11 @@ Look up the latest SDK version here: https://github.com/aws/aws-iot-device-sdk-j
 ## Samples App
 [Android IoT Samples App README](../samples/Android/README.md)
 
+## PKCS#11
+Connecting using PKCS#11 requires a PKCS#11 library which the user must supply. There are requirements the library must meet:
+* The PKCS#11 library **must** be compiled for Android and for use on the architecture of the target device.
+* The PKCS#11 library **must** have access to the PKCS#11 compatible smart card or Hardware Security Module (HSM) for storage and access to the private key file.
+* The path to the library needs to be provided to the builder for it to load and use the PKCS#11 library.
```

Comment:
"The builder" here is the build system, like gradle or maven?

Reply:
Good call, I'll provide a link to the builder we use for PKCS11